src/plugins/kibana/public/visualize/styles/main.less

```diff
@@ -2,21 +2,247 @@
 @import (reference) "~bootstrap/less/list-group";
 @import (reference) "~ui/styles/list-group-menu";
 
+@icon-background-color: #416376;
```

Comment:
Also, none of these styles are scoped in any way, do we really need special styling for the saved object loader just for this page? Can you work with @alt74 to find a style that works both in pages and in the drop downs?